extensions/keep-frontmost/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "keep-frontmost",
+  "title": "Keep Frontmost",
+  "description": "Closes All open apps except the frontmost one",
+  "icon": "Icon.png",
+  "author": "Mmehdi06",
+  "categories": [
+    "System"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "index",
+      "title": "Keep Frontmost",
+      "subtitle": "Close All Apps Except Frontmost",
```

Comment:
Subtitle should not be used to describe, can we add a better title?

Reply:
what about this?

title: Focus on active app
subtitle: SingleFocus

Reply:
yes you are right, i also thought about "Clear Distractions"

Reply:
It realy is I'll think about it further and come back to you later today. Thank you for the feedback.